webpack.config.js

```diff
@@ -36,9 +40,15 @@ module.exports = {
     resolve: {
         modules: [
             path.resolve(__dirname, "socialhome"),
+            path.resolve(__dirname, "mocha/fixtures"),
             path.resolve(__dirname, "node_modules")
         ],
-        alias: {vue: "vue/dist/vue.js",}
+        alias: {
+            "vue$": "vue/dist/vue.esm.js",
+            "bootstrap-vue": "bootstrap-vue/dist/bootstrap-vue.esm.js",
+            "fixtures": path.resolve(__dirname, "mocha/fixtures"),
```

Comment:
Now these are probably wrong - and probably unnecessary if the tests are working even if this is wrong ;)